src/commands/guild/userinfo.ts

```diff
@@ -0,0 +1,46 @@
+import { SlashCommandBuilder } from "@discordjs/builders";
+import { CommandInteraction, EmbedBuilder } from "discord.js";
+import { Command } from "../../types/command";
+import { LogLevelColor } from "../../utils/log";
+import { PermissionLevel } from "../../utils/permissions";
+
+const UserInfo: Command = {
+    permissionLevel: PermissionLevel.MEMBER,
+
+    data: new SlashCommandBuilder()
+        .setName("userinfo")
+        .setDescription("Displays the user information of the message author."),
+
+    async execute(interaction: CommandInteraction) {
+        const user = interaction.options.getUser("user") ?? interaction.user;
+        const name = `${user.username}#${user.discriminator}`;
+        const avatar = user.avatarURL() ?? "";
+
+        // The EMBED
+        const embed = new EmbedBuilder()
+            .setColor(LogLevelColor.INFO)
+            .setTitle(user.username)
+            .setAuthor({
+                name: name,
+                iconURL: avatar,
+            })
+            .setDescription(`Your ID: ${user.id}`)
+            .setThumbnail(avatar)
+            .addFields(
+                {
+                    name: "Joined Discord",
+                    value: user.createdAt.toDateString(),
```

Comment:
Can this use a Discord timestamp? https://gist.github.com/LeviSnoot/d9147767abeef2f770e9ddcd91eb85aa